packages/waku/src/server.ts

```diff
@@ -149,3 +146,26 @@ export function unstable_getHeaders(): Record<string, string> {
     string
   >;
 }
+
+type PlatformObject = {
+  buildData?: Record<string, unknown>; // must be JSON serializable
+  buildOptions?: {
+    deploy?:
+      | 'vercel-static'
+      | 'vercel-serverless'
+      | 'netlify-static'
+      | 'netlify-functions'
+      | 'cloudflare'
+      | 'partykit'
+      | 'deno'
+      | 'aws-lambda'
+      | undefined;
+  };
+} & Record<string, unknown>;
```

Comment:
What is the purpose of extending this with `Record<string, unknown>` when `buildData` is also present?

I guess I am just curious what this extra flexibility on the type might be used for. Currently it just looks like `buildData` is all that's being used.

Reply:
It's for #827 kind of extensions. It's not just for build.